src/lib/viewers/__tests__/BaseViewer-test.js

```diff
@@ -278,6 +278,36 @@ describe('lib/viewers/BaseViewer', () => {
             expect(document.defaultView.addEventListener).to.be.calledWith('resize', base.debouncedResizeHandler);
             expect(base.addListener).to.be.calledWith('load', sinon.match.func);
         });
+
+        it('should load the annotator when load is emitted with scale', (done) => {
+            sandbox.stub(fullscreen, 'addListener');
+            sandbox.stub(document.defaultView, 'addEventListener');
+            sandbox.stub(base, 'loadAnnotator');
+            base.annotationsPromise = {
+                then: () => {
+                    done();
+                }
+            };
+
+            base.addCommonListeners();
```

Comment:
The expectation is that the annotations promise is called. done() is only resolved when the annotations promise is called. Unfortunately we cannot expect base.loadAnnotator to be called since events are async.

I've added a check to ensure that base.scale is set and that base.loadAnnotator is passed into the promise.then function.